Digital-launcher.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec "/app/lib/python" "-OO" "/app/Digital.py" "$@"
```

Comment:
Ok, I have come across another problem on putting Digital.py on /app/bin

```python

# Given the Ren'Py base directory (usually the directory containing
# this file), this is expected to return the path to the common directory.
def path_to_common(renpy_base):
    return renpy_base + "/common"
```

Basically the Digital.py expects to be in the same place as the rest of the game's files and, if I want to use it to launch the game, I would have to patch the directories it tries to look.
It might be a better idea to use only the launcher on /app/bin, so I will go back to that.